json_object.c

```diff
@@ -81,11 +81,12 @@ static void json_object_fini(void)
 		if (json_object_table->count)
 		{
 			MC_DEBUG("json_object_fini: %d referenced objects at exit\n",
-			   json_object_table->count);
+				 json_object_table->count);
 			lh_foreach(json_object_table, ent)
 			{
 				struct json_object* obj = (struct json_object*)lh_entry_v(ent);
-				MC_DEBUG("\t%s:%p\n", json_type_to_name(obj->o_type), obj);
+				MC_DEBUG("\t%s:%p\n",
+					 json_type_to_name(obj->o_type), obj);
```

Comment:
Regardless of whether the changes are minor or not, if the only reason for changing a particular block of code is simply to adjust the formatting then that should go in a separate commit.
It's ok for now, since the changes are so small, but please keep them separate in the future.